desktop/extensions/marketplace.md

```diff
@@ -16,7 +16,7 @@ To install an extension:
 2. From the Dashboard, select **Add Extensions** in the menu bar.
    The Extensions Marketplace opens on the **Browse** tab.
 3. Browse the available extensions.
-   You can sort the list of extensions by **Recently added** or alphabetically.
+   You can sort the list of extensions by **Recently added**, **Most installed** or alphabetically. Alternatively, use **Filters** to search for extensions by category or whether they have been reviewed or not.
```

Comment:
```suggestion
   You can sort the list of extensions by **Recently added**, **Most installed**, or alphabetically. Alternatively, use **Filters** to search for extensions by category or whether they have been reviewed or not.
```

I think we use the serial comma, so might as well add it here? Also, maybe "alphabetically" should be in bold as none of these are direct references to UI elements.